release-notes/major10/10.0.0/migration.md

```diff
@@ -2,14 +2,17 @@
 
 This page describes how to update Valtimo from the previous version to the current.
 
-* **Breaking change 1/Deprecation 1**
+* **Java 17**
```

Comment:
Please add that this is meant for backend. Kind of a given in this case, but MS specifically asked for this and we as a team agreed to do this. Something like "Scope: Backend" just below the Java 17 bullet.